Source/WebCore/css/StyleProperties.cpp

```diff
@@ -333,7 +333,9 @@ String StyleProperties::getPropertyValue(CSSPropertyID propertyID) const
     case CSSPropertyContainer:
         if (propertyAsValueID(CSSPropertyContainerType) == CSSValueNormal)
             return getPropertyValue(CSSPropertyContainerName);
-        return getShorthandValue(shorthand, " / ");
+        return makeString(getPropertyValue(CSSPropertyContainerName), " / ", getPropertyValue(CSSPropertyContainerType));
```

Comment:
That’s what the line of code above does. If "type is normal, return just the name".

Reply:
By the way, while working on this I noticed many shorthands where we do not yet implement the shortest rule. I implemented many of those, it made the patch too large, so I cut down my "shortest" logic to only the amount needed to avoid a regression for now.

Reply:
Ah, sure